docs/rfc/block-registration.md

```diff
@@ -331,6 +331,36 @@ Block type editor style definition. It will only be enqueued in the context of t
 
 Block type frontend style definition. It will be enqueued both in the editor and when viewing the content on the front of the site.
 
```

Comment:
All block metadata fields are also included in the example on the top of the file for a better overview:
https://github.com/WordPress/gutenberg/blob/master/docs/rfc/block-registration.md#registering-a-block-type

Reply:
Can block define both `context` and `providesContext`? I assumed it can.